tests/sentry/api/endpoints/test_organization_code_mapping_details.py

```diff
@@ -57,6 +68,11 @@ def test_basic_edit(self):
         assert resp.data["id"] == str(self.config.id)
         assert resp.data["sourceRoot"] == "newRoot"
 
+    def test_basic_edit_from_member_permissions(self):
+        self.login_as(user=self.user2)
+        resp = self.make_put({"sourceRoot": "newRoot"})
+        assert resp.status_code == 200
```

Comment:
This is updating an existing code mapping:
`source_root: "/source/root" -> "newRoot"`

This is the code mapping:
https://github.com/getsentry/sentry/blob/c605f45a3ca0e585cd7ca1c5a0c286c0d61913e3/tests/sentry/api/endpoints/test_organization_code_mapping_details.py#L38-L45